tracer/src/Datadog.Tracer.Native/cor_profiler.cpp

```diff
@@ -1299,6 +1299,37 @@ void CorProfiler::RegisterIastAspects(WCHAR** aspects, int aspectsLength)
     }
 }
 
+void CorProfiler::UpdateSettings(WCHAR* keys[], WCHAR* values[], int length)
+{
+    for (int i = 0; i < length; i++)
+    {
+        if (WSTRING(keys[i]) == WStr("DD_TRACE_DEBUG"))
```

Comment:
It would be more efficient (less allocations) if you create a `WSTRING` out of the `DD_TRACE_DEBUG` and reuse it:

```
WSTRING debugVarName = WStr("DD_TRACE_DEBUG");

for (int i = 0; .....)
{
   if (keys[i] == debugVarName)
   {
    ....
   }
}
```